go.mod

```diff
@@ -5,14 +5,14 @@ go 1.16
 require (
 	github.com/go-logr/logr v0.4.0
 	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.13.0
+	github.com/onsi/gomega v1.17.0
```

Comment:
How did you choose this version? I checked [go.mod of csi-test](https://github.com/kubernetes-csi/csi-test/blob/5b2556c1c1ae81346a24b801b1f5bea11f9d3906/go.mod#L12-L13) following maintenance.md, and found csi-test is using ginkgo v1.14.2 and gomega v1.10.4.  Or just updating to latest release is enough for this time?

Reply:
The pvc-autoscaler isn't a CSI driver, so the pvc-autoscaler doesn't need dependent on the csi-test. So I maybe chose the latest release for ginkgo and gomega.

Reply:
That means [this instruction](https://github.com/topolvm/pvc-autoresizer/blob/5208b844d51210aa86ecc8b14c64c28c7833fca1/docs/maintenance.md?plain=1#L28) is not true for pvc-autoresizer? Or are there other version indications of ginkgo/gomega elsewhere other than `go.mod`?

Reply:
> That means this instruction is not true for pvc-autoresizer?

Yes, sorry.
I'll remove this section.

Reply:
No need to apologize :) Thank you fixing.